projects/rero/ng-core/src/lib/record/record.service.ts

```diff
@@ -153,8 +153,8 @@ export class RecordService {
     return this._http
       .delete<void>(this._apiService.getEndpointByType(type, true) + '/' + pid)
       .pipe(
-        tap(() => this.onDelete.next(this.createEvent(type, { pid }))),
-        catchError((error) => this.handleError(error))
+        catchError((error) => this.handleError(error)),
+        tap(() => this.onDelete.next(this.createEvent(type, { pid })))
```

Comment:
`createEvent` is private you can prefix with `_`

Reply:
done